hapi-fees/src/main/java/com/hedera/services/pricing/BaseOperationUsage.java

```diff
@@ -106,6 +109,13 @@ UsageAccumulator baseUsageFor(HederaFunctionality function, SubType type) {
 					case DEFAULT:
 						throw new IllegalArgumentException("Canonical usage unknown");
 				}
+			case TokenBurn:
+				switch (type) {
+					case TOKEN_NON_FUNGIBLE_UNIQUE:
```

Comment:
Do we also need to add the case for `TOKEN_FUNGIBLE_COMMON` sub type?

Reply:
@ljianghedera , For `TOKEN_FUNGIBLE_COMMON` it is going to be a fixed price and doesn't depend on the number of serial Numbers. So, no change need to be made for that scenario.

Reply:
I see. I was looking from the view point of `canonical-prices.json` file's point of view. For `TOKEN_FUNGIBLE_COMMON`, the resources used are fixed regardless how much tokens to burn.